grails-app/views/catalog.gsp

```diff
@@ -0,0 +1,208 @@
+                                        <img src='${resource(dir: "images/", file: "Grails.jpg")}' alt="Image Thumbnail" class="img-thumbnail">
+                                    </div>
+                                    <div class="col-md-6 col-xs-4 description">
+                                        <h5>Grails Framework<br />
+                                            <small>
+                                                Grails is an open source web application framework that uses the Apache Groovy programming language (which is in turn based on the Java platform).
+                                                It is intended to be a high-productivity framework by following the "coding by convention" paradigm,
+                                                providing a stand-alone development environment and hiding much of the configuration detail from the developer.
+                                            </small>
+                                        </h5>
+                                    </div>
+                                    <div class="col-md-2 col-xs-2">
+                                        <button class="btn btn-icon btn-danger"><i class="fa fa-plus"></i></button>
+                                        <br/>
+                                        <br/>
+                                        <a href="#" class="btn btn-info btn-fill" role="button">View</a>
+                                    </div>
+                                </div>
+                            </li>
+
+
+                        </ul>
+
+                        <div class="text-missing">
+                            <h5 class="text-muted">If you are not finding who you’re looking for try using an email address. </h5>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+</div>
+<footer class="footer-demo section-white-gray">
+    <div class="container">
+        <nav class="pull-left">
+            <ul>
+
+                <li>
+                    <a href="/">
+                        Groovy School
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        About
+                    </a>
+                </li>
+                <li>
+                    <a href="/catalog">
+                        Catalog
+                    </a>
+                </li>
+                <li>
+                    <a href="#">
+                        Contact
+                    </a>
+                </li>
+            </ul>
+        </nav>
+        <div class="copyright pull-right">
+            &copy; 2019 - GRD8I
+            <a href="#" target="_blank"><i class="fa fa-twitter"></i></a>
+            <a href="#" target="_blank"><i class="fa fa-facebook"></i></a>
+        </div>
+    </div>
+</footer>
+
+
+</body>
+
+
+<script>
```

Comment:
oh, you don't need any facebook pixel code. we won't be running any facebook adds in the nearest future. Please, could you remove this script section?